app/src/main/java/com/android/streetworkapp/utils/DateUtils.kt

```diff
@@ -29,3 +29,27 @@ fun String.toTimestamp(): Timestamp {
   val date = sdf.parse(this)
   return Timestamp(date!!)
 }
+
+/**
+ * Convert an epoch timestamp (Long) to a formatted string of format "dd/MM/yyyy HH:mm".
+ *
+ * @return The formatted string.
+ */
+fun Long.toFormattedString(): String {
+  val pattern = "dd/MM/yyyy HH:mm"
```

Comment:
Suggestion: since the string "dd/MM/yyyy HH:mm" is used now multiple times in the code, we could define a constant for it.